pallets/proposals/src/impls.rs

```diff
@@ -553,30 +564,41 @@ impl<T: Config> Pallet<T> {
                     }
                 });
 
-        let available_funds: BalanceOf<T> = unlocked_funds - project.withdrawn_funds;
+        let withdrawable: BalanceOf<T> = unlocked_funds - project.withdrawn_funds;
+
         ensure!(
-            available_funds > (0_u32).into(),
+            withdrawable > (0_u32).into(),
             Error::<T>::NoAvailableFundsToWithdraw
         );
 
+        let fee = withdrawable.saturating_mul(<T as Config>::ImbueFee::get().into())
+            / MAX_PERCENTAGE.into();
+        let withdrawn = withdrawable.saturating_sub(fee);
+
+        let project_account = Self::project_account_id(project_key);
+        let pallet_account = Self::account_id();
+
+        // Take the fee
+        T::MultiCurrency::transfer(project.currency_id, &project_account, &pallet_account, fee)?;
```

Comment:
Perhaps the fee should go to the treasury instead, the pallet might be good enough for now though
@samelamin what do you think

Reply:
Also should fees be in IMBU or the project currency

Reply:
The fee should go to the treasury but im happy to separate where it goes into its own pr 

Fees are 5% of the target currency, lets keep it simple since we already require deposits in $IMBU